tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2608,7 +2645,39 @@ private String handleDateFormatItem(String xpath, String value, boolean showCont
                 DateFormatSymbols dfs = sdf.getDateFormatSymbols();
                 dfs.setTimeSeparatorString(timeSeparator);
                 sdf.setDateFormatSymbols(dfs);
-                if (id == null || id.indexOf('B') < 0) {
+                if (parts.contains("availableFormats")) {
+                    String example = addExampleResult(sdf.format(DATE_SAMPLE) + ", " + sdf.format(DATE_SAMPLE5), "", showContexts);
```

Comment:
I should only do this when `sdf.format(DATE_SAMPLE) != sdf.format(DATE_SAMPLE5)`